Assets/Scripts/Features/Coins/CoinsVisual.cs

```diff
@@ -62,6 +62,7 @@ void Update()
                         var adding = Feature.VolumeAmounts[coin.Volume - 1];
                         Feature.Wallet.AddToWallet(adding).Forget();
                         coin.Pick();
+                        Feature.FloatingText.PopText(coin.transform.position, coin.Volume.ToString(),2f);
```

Comment:
The Volume of the Coin is the Size of it. 
The amount of Money is Feature.VolumeAmounts[coin.Volume - 1]

Reply:
Oh I see thanks